core/src/main/java/org/mskcc/cbio/portal/util/MskWholeSlideViewerTokenGenerator.java

```diff
@@ -9,25 +9,23 @@
 	// ~ Static fields
 	// ========================================================================================================
 
+    public String hash = null;
+    public String timeStamp = String.valueOf(System.currentTimeMillis());
 	public static final String PORTAL_NAME = "cbioportal";
 
 	// ~ Methods
 	// ========================================================================================================
 
-	public static String generateTokenByHmacSHA256(String username, String secretKey) {
+	public void generateTokenByHmacSHA256(String username, String secretKey) {
```

Comment:
@dionnezaal why does the signature of this method have to change? Any specific reason?